frame-metadata/src/lib.rs

```diff
@@ -172,6 +176,12 @@ pub enum RuntimeMetadata {
 	/// Version 14 for runtime metadata, as raw encoded bytes.
 	#[cfg(not(feature = "v14"))]
 	V14(OpaqueMetadata),
+	/// Version 15 for runtime metadata.
+	#[cfg(feature = "v15-unstable")]
+	V15(v15::RuntimeMetadataV15),
```

Comment:
I didn't notice, but how do you ask for V15 metadata in Substrate; will you ask for metadata_at_version(u32::MAX) or something? Or do you ask for 15? Just wondeirng whether we do anything to highlight the instability of it in the code